tests/tcp.rs

```diff
@@ -28,3 +32,53 @@ pub fn should_work_tcp_stream() {
     assert!(!result.is_read(&stream));
     assert!(result.is_write(&stream));
 }
+
+#[test]
+pub fn should_work_with_multiple_fds() {
+    let mut selector = Selector::new();
+
+    let server = thread::spawn(|| {
+        let listener = TcpListener::bind("localhost:1234").expect("Couldn't bind to the address...");
```

Comment:
maybe try `127.0.0.1` directly instead of localhost?
Not sure if github CI somewhat different from regular system